app/build.gradle

```diff
@@ -1,8 +1,12 @@
+plugins {
```

Comment:
Is JPL using SonarQube? 

Does this require a contributor to have a `SonarQube` server running locally to add value?

Reply:
Along with learning how to build a rover, having SonarQube server running locally would also help to write clean code